lib/babel.js

```diff
@@ -28,3 +28,9 @@ var transpiled = babel.transformFileSync(testPath, options);
 requireFromString(transpiled.code, testPath, {
 	appendPaths: module.paths
 });
+
+process.on('message', function (message) {
+	if (message['ava-kill-command']) {
+		process.exit(0);
```

Comment:
It wouldn't really ever make sense for them to do that though (not unless they fork an additional process).